core/rawdb/database_quorum.go

```diff
@@ -81,28 +86,39 @@ func GetPrivateBlockBloom(db ethdb.Database, number uint64) (bloom types.Bloom)
 	return bloom
 }
 
-type PrivacyMetadataLinker interface {
-	PrivacyMetadataRootForPrivateStateRoot(privateStateRoot common.Hash) common.Hash
-	LinkPrivacyMetadataRootToPrivateStateRoot(privateStateRoot, privacyMetadataRoot common.Hash) error
+// AccountExtraDataLinker maintains mapping between root hash of the state trie
+// and root hash of state.AccountExtraData trie
+type AccountExtraDataLinker interface {
+	// Find returns the root hash of the state.AccountExtraData trie from
+	// the given root hash of the state trie.
+	//
+	// It returns an empty hash if not found.
+	Find(stateRoot common.Hash) common.Hash
```

Comment:
MINOR: I would rename 
- `Find` --> `GetAccountExtraDataRoot`
- `Link` --> `WriteAccountExtraDataRoot`

Would be clearer to understand in the rest of the code

Reply:
`Find` --> `GetAccountExtraDataRoot` makes sense. Changed.
I would keep `Link` function as it reflects the interface's responsibility